docs/faq/errors.md

```diff
@@ -0,0 +1,46 @@
+# Errors and Common Problems #
+
+## **Paymetheus** ##
+
+#### **1. I've just started using Paymetheus, but it seems to be stuck.** ####
+
+The first time you start Paymetheus, it will download the blockchain. this can take up
+to an hour and Paymetheus will appear to be doing nothing. This will only happen once.
+
+#### **2. Invalid passphrase for master private key.** ####
+
+This is just a long way of saying, "Incorrect password". You entered the wrong password for your
+wallet.
+
+#### **3. "Unable to purchase tickets: insufficient funds available..." but the wallet says I have enough.** ####
+
+There is a known bug in Paymetheus where immature funds as counted as available. After a ticket votes, there is a 
+256 block window where the funds are still locked. In this state, they are known as immature. When the period expires
+they will be usable again.
+
+#### **4. Paymetheus is displaying the wrong balance.** ####
+
+If Paymetheus displays the wrong balance, you can fix it by using the command line utility to overwrite some files.
+Some of this can be confusing if you are not familiar with the command line, but just follow the instructions line by line
+and you'll be fine. Where you see commands that `look like this`, just copy them and paste them exactly as they are into the command line.
+Don't forget to press <ENTER> after each command. If you get stuck, join our [Slack channel](https://decred.slack.com) and someone will help,
+But first, check the command and make sure you entered it correctly. You will need your wallet seed. This process should take about 15 minutes.
+
+1. We're going to open three PowerShell windows. Press the Window key. Type 'powershell' (don't type the quotes here or in the future) and press enter.
+2. Do this two more times.
+3. Move the windows so you can see all of them.
+4. Copy and paste the following command: `cd $env:programfiles/decred/paymetheus` (Note, in powershell, just right click to paste). Press <ENTER>.
+5. Run the same command in the other two windows.
+6. Open Windows Explorer.
+7. Paste `%localappdata%/decred/paymetheus` into the location bar. Press Enter.
+8. Delete the 'mainnet' folder.
+9. Go to one of the PowerShell windows and paste `./dcrd -u you -P sekrit`. Press <ENTER>.
```

Comment:
people will ask what these are. add a note here that they are the rpc username and password and can be anything